images/cfw/usr/etc/system/webhmi/little_webhmi.py

```diff
@@ -0,0 +1,112 @@
+# aiohttp variant
+import aiohttp
+from aiohttp import web
+import asyncio
+import subprocess
+import json
+import threading
+import aiohttp_jinja2
+import jinja2
+import os
+
+# Print the current working directory for debugging
+print("Current working directory:", os.getcwd())
+
+# Global variable to hold the current job name
+current_job_name = "None"
+# Global variable to hold the latest message
+latest_message = ""
+
+async def update_current_job():
+    global current_job_name
+    while True:
+        try:
+            with open('/userdata/print_ctx.json', 'r') as file:
+                data = json.load(file)
+                subtask_name = data['subtask_name']
+                current_job_name = subtask_name
+        except Exception as e:
+            print(f"Error reading or parsing JSON file: {e}")
+        await asyncio.sleep(10)
```

Comment:
Can we get these data either from DBus or MQTT instead of having to poll?  Are these data broadcast on DBus as a signal?